tests/integration/helpers/factories/models/invenio_records.py

```diff
@@ -27,13 +27,20 @@
 from invenio_records.models import RecordMetadata
 
 from helpers.factories.models.base import BaseFactory
-from helpers.factories.providers.faker import faker
+from helpers.providers.faker import faker
 
 
 class RecordMetadataFactory(BaseFactory):
     class Meta:
         model = RecordMetadata
 
+    @classmethod
+    def _adjust_kwargs(cls, **kwargs):
```

Comment:
something more explicit would be nicer, like `_update_data_in_kwargs`

Reply:
https://factoryboy.readthedocs.io/en/latest/reference.html?highlight=_adjust_kwargs#factory.Factory._adjust_kwargs